### ISSUE TITLE
Navigation from List view broken

### ISSUE THREAD

**Issue by nitina32** [frappe/erpnext#32819]
### Information about bug

Hi
After updating to the latest frappe version i.e 14.13.0 & Erpnext 14.4.0, I am facing an issue of broken navigation from list view to the record page.
Upon investigation I realize that all my records have primary ids with character “/” in them which are not effective masked to ascii “%2F” when being input in the URL, thus breaking navigation.

```
eg:
current url:
https://www.xyz.com/app/journal-entry/JE/2223/0271

masked:
https://www.xyz.com/app/journal-entry/JE%2F2223%2F0271
```

This is more or less specific to navigation from list view. When clicking on record id on GL report the system is masking and navigating correctly (fails incase of stock ledger though).

Thanks
Nitin
ps. even after updating to Erpnext v14.5.0 it doesn't resolve the issue

### Module

accounts, other

### Version

Frappe version - 14.14.0
ERPNEXT - 14.5.0

**Comment by chechani** [comment]
we too have same issue in case of Purchase Orders, Purchase Receipt and Purchase Invoices

**Comment by chechani** [comment]
solved via this PR
https://github.com/frappe/frappe/pull/17092

**Comment by chechani** [comment]
update available in frappe v 14.4.2
